tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_2017_gfdlmp_regional_plot.yaml

```diff
@@ -12,8 +12,9 @@ workflow:
   DATE_LAST_CYCL: '2019070100'
   FCST_LEN_HRS: 6
   PREEXISTING_DIR_METHOD: rename
-workflow_switches:
-  RUN_TASK_PLOT_ALLVARS: true
+rocoto:
+  tasks:
+    taskgroups: '{{ ["parm/wflow/prep.yaml", "parm/wflow/coldstart.yaml", "parm/wflow/post.yaml", "parm/wflow/plot.yaml"]|include }}'
```

Comment:
Is ther reason why plot task needed this because we can't put a task entry in the xml file unless it is used now?
I think the lack of jinja templating in the yaml template files (not the support we have inside dictionary entries but for a "block" of code) is restrictive compared to the xml template. In my opinion, specifying a list of taskgroups in the user config files looks cumbersome for me compared having flags to test if the task should be run.

Reply:
I'm not sure I understand your concern. The taskgroups entry specifies which tasks to run. There is not a mechanism currently to "add to the base tasks" other than just define the whole list of groups of tasks that should run. 

In  say that this is far less restrictive in the sense that we can literally define any task to run with this method, not just those already enabled by pre-defined templates.

Reply:
I was talking about the lack of jinja templating in the workflow yaml files under `param/wflow` such as `default_workflow.yaml` being restrictive. I am not sure if this is be design, for example, to avoid the workflow yaml files from going down the path of the previous approach of using jinja templates and becoming unreadable.
On that note, I need your feedback on this commit where I added the rrfs prep tasks, assuming jinja templating is implemented in the workflow yaml files. 
https://github.com/ufs-community/ufs-srweather-app/pull/647/commits/a6bde3447e9f4a8d572e27001dcc7775e6952f8b

With regard to defining `taskgroups` in the user config files is what I refered to as cumbersome. I think it would be easier for the user to just specify whether to run a task or not using the old `workflow_switches:` set of variables or other logical variables that enable a whole set of tasks. Depending on that, we can figure out what workflow yaml files to include to compose the whole workflow hidden to the user.